e2e/specs/create.task.spec.js

```diff
@@ -0,0 +1,25 @@
+const SingIn = require('../pages/SignIn');
+let username = 'kevinherrera2';
+let password = '70723844';
+let projectID = '2203114';
+describe('pivotal tracker project page add task to story', () => {
+    let project;
+    let dashboard;
+    let story;
+    let storyTasks;
+    let taskList = ['task1', 'task2', 'task3'];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "taskList" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'taskList' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)